liberty-support-feature/JakartaEE9_README.md

```diff
@@ -40,7 +40,7 @@ Example:
       <artifactItem>
         <groupId>io.openliberty.arquillian</groupId>
         <artifactId>arquillian-liberty-support-jakarta</artifactId>
-        <version>2.1.3</version>
+        <version>3.0.0</version>
```

Comment:
change to `3.0`

Reply:
When I look at maven, it has 2.0.0 or 2.1.0 for the version number.  In the past for 2.1 initial version, this was 2.1.0 in the README.  As such I assumed it will be 3.0.0 and not just 3.0.

Reply:
If you want it to be `3.0.0`, can you change the pom.xml version numbers to match with this PR?